src/jsMain/kotlin/net/kyori/adventure/webui/js/ExternalFunctions.kt

```diff
@@ -17,3 +18,6 @@ public external class bulmaToast {
         public fun setDefaults(settings: Json)
     }
 }
+public fun bulmaToast.Companion.toast(message: String, type: String) {
```

Comment:
Should have a newline here. Also, let's make `type` default to `"is-success"`.

Reply:
Okay! I was unsure if `is-success` or `is-info` should be default